dev/src/dev/render_png.clj

```diff
@@ -87,9 +88,7 @@
 
 (comment
   (render-card-to-png 1)
-
-  (let [{:keys [content]} (render-pulse-card 1)]
-    (open-hiccup-as-html content))
+  (render-card-to-png 2700)
```

Comment:
I don't think this change is adding anything, the old version was there to help you see how to use this stuff.

Reply:
Sorry, this was a really old PR I should have closed. The new version is [here]([url](https://github.com/metabase/metabase/pull/43555)) (still a wip progress)